GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs

```diff
@@ -131,18 +132,20 @@ private void InitializeComponent()
             tableLayoutPanel1.Controls.Add(this.comboMinWidthLabel, 0, 5);
             tableLayoutPanel1.Controls.Add(this.sortLessRecentRepos, 0, 3);
             tableLayoutPanel1.Controls.Add(this.shorteningGB, 0, 4);
+            tableLayoutPanel1.Controls.Add(this.showBranchName, 0, 7);
             tableLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Left;
             tableLayoutPanel1.Location = new System.Drawing.Point(0, 0);
             tableLayoutPanel1.Name = "tableLayoutPanel1";
             tableLayoutPanel1.Padding = new System.Windows.Forms.Padding(8);
-            tableLayoutPanel1.RowCount = 7;
+            tableLayoutPanel1.RowCount = 8;
             tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 20F));
+            tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
```

Comment:
These two lines might have to be swapped.
Could you add screenshots of this form, please?

Reply:
Sure here you go
![grafik](https://user-images.githubusercontent.com/7032094/77233998-8f8a3300-6bab-11ea-8e2e-d0f8baf25f6d.png)